src/status_im/chat/events/commands.cljs

```diff
@@ -24,26 +24,26 @@
 (defn request-command-message-data
   "Requests command message data from jail"
   [db
-   {{command-name         :command
+   {{command-name :command
      content-command-name :content-command
-     :keys                [content-command-scope
-                           scope
-                           params
-                           type
-                           bot]} :content
-    :keys                        [chat-id jail-id group-id] :as message}
+     :keys [content-command-scope scope params type bot]} :content
+    :keys [chat-id jail-id group-id] :as message}
    data-type]
   (let [{:keys          [chats]
          :accounts/keys [current-account-id]
          :contacts/keys [contacts]} db
-        jail-id (or bot jail-id chat-id)
-        jail-id (if (get-in chats [jail-id :group-chat])
-                  (get-in chats [jail-id :possible-commands (keyword command-name) :owner-id])
-                  jail-id)]
+        jail-id               (or bot jail-id chat-id)
+        jail-command-name     (or content-command-name command-name)
+        ;; here we're trying to use the default scope if there is no other scope provided
+        default-command-scope (-> (get-in contacts [jail-id :commands (keyword jail-command-name)])
```

Comment:
`(get-in contacts [jail-id :commands (keyword jail-command-name) 0 :scope])` ?

Reply:
It works for vector, but not for lists.
```
cljs.user=> (def a (list 1 2 3 4))
#'cljs.user/a
cljs.user=> (get a 0)
nil```